lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -101,6 +100,8 @@ def trigger
       puppet_trigger['start_date'] = self.class.normalized_date("#{trigger['start_year']}-#{trigger['start_month']}-#{trigger['start_day']}")
       puppet_trigger['start_time'] = self.class.normalized_time("#{trigger['start_hour']}:#{trigger['start_minute']}")
       puppet_trigger['enabled']    = trigger['flags'] & Win32::TaskScheduler::TASK_TRIGGER_FLAG_DISABLED == 0
+      puppet_trigger['minutes_interval'] = trigger['minutes_interval'] ||= 0
+      puppet_trigger['minutes_duration'] = trigger['minutes_duration'] ||= 0
```

Comment:
Need to debug a bit more, but I'm seeing what @joshcooper had suggested (your output above shows the same for `GoogleUpdateTaskMachineUA`) -  `'minute_duration' => '1440'`.  The task scheduler UI describes this as a `Daily At 1:28 PM every day` trigger

```puppet
scheduled_task { 'GoogleUpdateTaskMachineUA':
  ensure    => 'present',
  arguments => '/ua /installsource scheduler',
  command   => 'C:\Program Files (x86)\Google\Update\GoogleUpdate.exe',
  enabled   => 'true',
  trigger   => [{'enabled' => 'true', 'every' => '1', 'index' => '0', 'minutes_duration' => '1440', 'minutes_interval' => '60', 'schedule' => 'daily', 'start_date' => '2015-2-6', 'start_time' => '13:28'}],
  user      => 'system',
}
```

Keep in mind that we don't presently support multiple triggers, and only report on the first one.  `GoogleTaskMachineCore` on my machine has 2 triggers, and the first is a `logon` style trigger, so that's why we see a `minutes_duration` of `0`